pages/404.tsx

```diff
@@ -5,10 +5,18 @@ import styles from "../styles/Home.module.css";
 const NotFound: NextPage = () => {
   return (
     <div className={styles.screen}>
-      <div className="flex items-center justify-center h-full">
-        <div className="text-center">
-          <h1 className="title">404 - Page Not Found</h1>
-          <p className="mt-2">The page you are looking for does not exist.</p>
+      <div className={styles.wrapperScreen}>
```

Comment:
<img width="735" alt="Capture d’écran 2024-11-24 à 21 31 38" src="https://github.com/user-attachments/assets/5f86a4b9-7341-40cc-9001-a9b7ce1546af">

You didn't test here, nothing has changed it's still not centered and in green.

is this class actually defined in `global.css`? The example I gave is another repo so you need to [see in this other repo how do we code it ](https://github.com/lfglabs-dev/app.starknet.id/blob/testnet/styles/globals.css)

Reply:
@fricoben the class is defined in global.css, yes. 
The thing is, it appears what is currently deployed is the brown-ish colour, but what is currently in the global.css for app.starknet.id is green

If the green is wrong, maybe you should open an issue for it to be worked on as well. 

For the centering, my bad it was an oversight. In the push I'm about to make I've changed it to the brown.